oktapam/resource_server_checkout_settings_test.go

```diff
@@ -0,0 +1,186 @@
+	description = "test resource group"
+	delegated_resource_admin_groups = [oktapam_group.test_resource_group_dga_group.id]	
+}
+resource "oktapam_resource_group_project" "test_acc_resource_group_project" {
+	name = "%s"
+	resource_group = oktapam_resource_group.test_acc_resource_group.id
+	ssh_certificate_type  = "CERT_TYPE_ED25519_01"
+	account_discovery     = true
+}
+resource "oktapam_server_checkout_settings" "test_acc_server_checkout_settings" {
+	resource_group = oktapam_resource_group.test_acc_resource_group.id
+	project = oktapam_resource_group_project.test_acc_resource_group_project.id
+	checkout_required = true
+	checkout_duration_in_seconds = 900
+}	
+`
+
+func createTestAccServerCheckoutSettingsCreateConfig(dgaName, resourceGroupName, projectName string) string {
+	return fmt.Sprintf(testAccServerCheckoutSettingsCreateConfigFormat, dgaName, resourceGroupName, projectName)
+}
+
+const testAccServerCheckoutSettingsUpdateConfigFormat = `
+provider "oktapam" {}
+resource "oktapam_group" "test_resource_group_dga_group" {
+	name = "%s"
+}
+resource "oktapam_resource_group" "test_acc_resource_group" {
+	name = "%s"
+	description = "test resource group"
+	delegated_resource_admin_groups = [oktapam_group.test_resource_group_dga_group.id]	
+}
+resource "oktapam_resource_group_project" "test_acc_resource_group_project" {
+	name = "%s"
+	resource_group = oktapam_resource_group.test_acc_resource_group.id
+	ssh_certificate_type  = "CERT_TYPE_ED25519_01"
+	account_discovery     = true
+}
+resource "oktapam_server_checkout_settings" "test_acc_server_checkout_settings" {
+	resource_group = oktapam_resource_group.test_acc_resource_group.id
+	project = oktapam_resource_group_project.test_acc_resource_group_project.id
+	checkout_required = true
+	checkout_duration_in_seconds = 3600
+	include_list = ["vaulted_account_1", "vaulted_account_2"]
+}	
+`
+
+func createTestAccServerCheckoutSettingsUpdateConfig(dgaName, resourceGroupName, projectName string) string {
+	return fmt.Sprintf(testAccServerCheckoutSettingsUpdateConfigFormat, dgaName, resourceGroupName, projectName)
+}
+
+const testAccServerCheckoutSettingsDeleteConfigFormat = `
+provider "oktapam" {}
+resource "oktapam_group" "test_resource_group_dga_group" {
+	name = "%s"
+}
+resource "oktapam_resource_group" "test_acc_resource_group" {
+	name = "%s"
+	description = "test resource group"
+	delegated_resource_admin_groups = [oktapam_group.test_resource_group_dga_group.id]	
+}
+resource "oktapam_resource_group_project" "test_acc_resource_group_project" {
+	name = "%s"
+	resource_group = oktapam_resource_group.test_acc_resource_group.id
+	ssh_certificate_type  = "CERT_TYPE_ED25519_01"
+	account_discovery     = true
+}
+`
+
+func createTestAccServerCheckoutSettingsDeleteConfig(dgaName, resourceGroupName, projectName string) string {
+	return fmt.Sprintf(testAccServerCheckoutSettingsDeleteConfigFormat, dgaName, resourceGroupName, projectName)
+}
+func TestAccServerCheckoutSettingsSource(t *testing.T) {
```

Comment:
Add some more test for below cases - 

1. First, removing exclude_list and changing it to include_list
2. Then, remove include_list and use just checkout_required field
3. Provide both include and exclude list
4. Update project/resource group